src/snippets/article-preprint.v1.yaml

```diff
@@ -0,0 +1,21 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Article pre-print version snippet
+type: object
+properties:
+    type:
```

Comment:
My observation is although `type` is probably matches the XML terminology, the value of "preprint" is similar to the `status` values like `poa` and `vor`. The snippet definitions can be like apples and oranges, and I don't feel strongly either way.

Reply:
From [this comment](https://github.com/elifesciences/issues/issues/4702#issuecomment-853699910-permalink) we can expect these as well:

* preprint
* accepted-manuscript
* version-of-record
* corrected-version-of-record

... although, we already have a version-of-record (`status: VOR`) 'event' and I guess a 'corrected-version-of-record' would simply be a VOR v2, v3, etc. And accepted-manuscript is POA. So ... I guess we can ignore those when they come along and @nlisgo's suggestion about making the preprint data top-level has more merit. 

I still think that now that there is an extension point in the XML it will be used. Just like we have articles with 5 or 6 VOR versions, I wouldn't be surprised to see an article with multiple preprint events.

Reply:
changed `type` to `status` for consistency.